### PR TITLE
HBW-219 Remove menu button

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: hbw/app/assets/stylesheets/hbw.sass [hunk 1]
```diff
@@ -634,62 +614,11 @@
   height: 60px
 
 #hbw-tasks
-  .hbw-disabled
-    visibility: hidden
-
   .hbw-enabled
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: It's used in `tass_list.js.jsx`, isn't it?

File: hbw/app/javascript/packs/hbw/widget.js.jsx [hunk 2]
```diff
@@ -61,13 +56,6 @@ modulejs.define(
 
       changeTask (task) {
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/widget.js.jsx]: https://github.com/latera/homs/pull/391/files#diff-2e271f00419ac5da37c3b151a10c6994L58
Are you sure it's still used?

File: hbw/app/javascript/packs/hbw/widget.js.jsx [hunk 3]
```diff
@@ -109,7 +80,7 @@ modulejs.define(
           this.renderClaimingTaskList(this.availableTaskListContainer);
         }
 
-        if (this.widget || this.tasksMenu) {
+        if (this.widget) {
           this.subscribeOnTasks();
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/widget.js.jsx]: BTW where do we use this subscription?

File: hbw/app/javascript/packs/hbw/widget.js.jsx (hunk 2 above)
Comment (mxsmirnoff) [reply hbw/app/javascript/packs/hbw/widget.js.jsx]: Link does not work.
But is used: 
```
this.env.dispatcher.bind('hbw:task-clicked', 'widget', this.changeTask);
```
And it is triggering in `entity_task/s.js.jsx`.

File: hbw/app/javascript/packs/hbw/widget.js.jsx (hunk 3 above)
Comment (mxsmirnoff) [reply hbw/app/javascript/packs/hbw/widget.js.jsx]: Nowhere, as I can see.

File: hbw/app/javascript/packs/hbw/widget.js.jsx (hunk 2 above)
Comment (TimAle) [reply hbw/app/javascript/packs/hbw/widget.js.jsx]: Yes, link navigated to this line :)
I see that it's triggering but I wonder why. I thought these events were about side menu only. Am I wrong?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+v2.2.0 [unreleased]
+-------------------
+### Refactoring
+- [#391](https://github.com/latera/homs/pull/391) Removed side menu with tasks.
```
Comment (TimAle) [inline CHANGELOG.md]: Removed -> Remove

**Pull request merged**